package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-page-object",
-  "version": "2.0.0-beta.2",
+  "version": "2.0.0-beta.3",
```

Comment:
There's a `beta.5` release that was released a few days ago that deals with some jQuery issues -maybe worth checking out?

https://github.com/san650/ember-cli-page-object/releases/tag/v2.0.0-beta.5

Reply:
Yes. Unfortunately it is a v2 addon, so I have to upgrade to ember-auto-import v2 first. And in order to do that, I have to upgrade ember-prop-types and ember-bootstrap. The good news is I hope this fork is temporary.